pkg/skaffold/schema/latest/v1/config.go

```diff
@@ -536,6 +539,15 @@ type DeployType struct {
 	KustomizeDeploy *KustomizeDeploy `yaml:"kustomize,omitempty"`
 }
 
+// DockerDeploy uses the `docker` CLI to create application containers in Docker.
+type DockerDeploy struct {
+	// UseCompose tells skaffold whether or not to deploy using `docker-compose`.
+	UseCompose bool `yaml:"useCompose,omitempty"`
```

Comment:
This shouldn't be enabled until we actually support it.  As is, it will show up in our JSON schema.